lang/fa.json

```diff
@@ -1,26 +1,26 @@
 {
   "Play": "پخش",
-  "Pause": "وقفه",
+  "Pause": "مکث",
   "Current Time": "زمان کنونی",
   "Duration Time": "مدت زمان",
   "Remaining Time": "زمان باقیمانده",
   "Stream Type": "نوع استریم",
   "LIVE": "زنده",
-  "Loaded": "فراخوانی شده",
+  "Loaded": "بارگیری شده",
   "Progress": "پیشرفت",
-  "Fullscreen": "تمام صفحه",
-  "Non-Fullscreen": "نمایش عادی",
+  "Fullscreen": "تمام‌صفحه",
+  "Non-Fullscreen": "غیر تمام‌صفحه",
   "Mute": "بی صدا",
-  "Unmute": "بهمراه صدا",
+  "Unmute": "با صدا",
   "Playback Rate": "سرعت پخش",
   "Subtitles": "زیرنویس",
   "subtitles off": "بدون زیرنویس",
-  "Captions": "عنوان",
+  "Captions": "عنوان‌ها",
   "captions off": "بدون عنوان",
```

Comment:
Should this also change like "captions"?

Reply:
Yes i think. i will review again and add all strings and will submit it again.
thanks.